### PR TITLE
[Y360CMS-216] Add live stream API endpoints

### PR DESCRIPTION
**Pull request by AndreyMaximov**
The PR adds 3 API methods for Live Streams:
- fetch live streams for a category
- search live streams
- fetch individual live stream

### CODE REVIEW

**Comment by bousquet** [comment]
@AndreyMaximov  How are timezones handled in the CMS?  Will each person who enters them use their local timezone to specify the start time?  Or does an association have a timezone setting we can use?

**Review by bousquet — APPROVED** [review]
See my comment about timezones but these api changes look good.  Are the livestreams returns all currently live or in the future?

**Comment by AndreyMaximov** [comment]
@bousquet each user account has the timezone setting

**Review by bousquet — CHANGES_REQUESTED** [review]
@AndreyMaximov is there an event status in this feed?  (pending, preparing, live, completed, etc.)

File: vc.adoc [hunk 1]
```diff
@@ -1958,6 +1989,388 @@ Content-Type: application/json;charset=UTF-8
...
+----
+
+
+[[_live_stream_details]]
+== link:#_live_stream_details[Live Stream Details]
+
+[[resources-fetch-live-stream]]
+=== link:#resources-fetch-live-stream[Fetch Live Stream]
+
+`GET /api/virtual-classes/v3.0/content-providers/{provider}/live-stream/{liveStreamId}`
```
Comment (ctmaloney) [inline vc.adoc]: This line is missing the 's' on live-streams should be:
GET /api/virtual-classes/v3.0/content-providers/{provider}/live-streams/{liveStreamId}

Comment (AndreyMaximov) [reply vc.adoc]: That's fixed

**Pull request merged**